src/createAsyncThunk.ts

```diff
@@ -60,20 +89,12 @@ export const miniSerializeError = (value: any): any => {
 export function createAsyncThunk<
   Returned,
   ThunkArg = void,
-  State = unknown,
-  Extra = unknown,
-  DispatchType extends Dispatch = Dispatch,
-  ActionType extends string = string,
-  ThunkAPI extends BaseThunkAPI<any, any, any> = BaseThunkAPI<
-    State,
-    Extra,
-    DispatchType
-  >
+  ThunkApiConfig extends AsyncThunkConfig = {}
 >(
-  type: ActionType,
```

Comment:
I've removed this type completely since it does not seem to have any value - all action types are concatenation to that string, so it'll never be used again.